src/Microsoft.AspNetCore.Server.IntegrationTesting/Common/DeploymentParameters.cs

```diff
@@ -106,6 +106,8 @@ public class DeploymentParameters
 
         public string PublishedApplicationRootPath { get; set; }
 
+        public HostingModel HostingModel { get; set; }
```

Comment:
Set a default value?

Reply:
Other values don't have default values (including enums). See: https://github.com/aspnet/Hosting/pull/1261/files#diff-8478a3aebbec4604e8f2cf79bb152195R105

Reply:
By default, it would take the first value as the default. Do we want to default all tests to InProcess?

Reply:
Probably should be out of process.

Reply:
I'm okay with changing the order of the enums but personally I prefer the more explicit approach where we just set the default value. Takes out all ambiguity.